podcasts/ShelfActionsViewController+Table.swift

```diff
@@ -18,10 +18,10 @@ extension ShelfActionsViewController: UITableViewDelegate, UITableViewDataSource
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         if tableView.isEditing {
             if section == ShelfActionsViewController.shortcutSection {
-                return Constants.Limits.maxShelfActions
+                return Constants.Limits.maxShelfActions + maxShelfActionsAdjustment
             }
 
-            return allActions.count - Constants.Limits.maxShelfActions
+            return allActions.count - ( Constants.Limits.maxShelfActions + maxShelfActionsAdjustment)
```

Comment:
⛏️ I think an extra space snuck in here

```suggestion
            return allActions.count - (Constants.Limits.maxShelfActions + maxShelfActionsAdjustment)
```